app/controllers/lookup_keys_controller.rb

```diff
@@ -5,7 +5,6 @@ class LookupKeysController < ApplicationController
 
   def index
     @lookup_keys = resource_base.search_for(params[:search], :order => params[:order])
-                                .includes(:param_classes)
```

Comment:
@eLobato yes it was moved here: https://github.com/theforeman/foreman/pull/2466/files#diff-0e73fac86975312550dc0afe8d868497R5